keps/sig-cloud-provider/20180530-cloud-controller-manager.md

```diff
@@ -352,7 +352,7 @@ common integration hooks may be added
 The development of the
 [Cloud Controller Manager](https://github.com/kubernetes/kubernetes/tree/master/cmd/cloud-controller-manager)
 and
-[Cloud Provider Interface](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/cloud.go)
```

Comment:
This can be https://github.com/kubernetes/cloud-provider/blob/master/cloud.go now :)

Reply:
+1

Reply:
The problem with embedding links to code in a KEP is that when that code inevitably moves, that KEP becomes broken.  There are a bunch of PRs open to fix links.  Some change to "master", some change to branches, and none with any particular rhyme or reason that I can discern.

I think we should just leave old KEPs alone.  My inclination is to close all of these PRs - tell me why I should not?